tests/test_TFNetworkLayer.py

```diff
@@ -388,6 +388,40 @@ def test_activation_layer_net_construct():
     assert_equal(v.shape, (n_batch, seq_len, num_inputs))
     assert_equal(v.tolist(), [[[0, 0], [0, 0], [2, 2]]])
 
+def test_activation_layer_abs_for_stft():
```

Comment:
Add two empty lines before a function.
(Please check our generic code style rules.)

Reply:
https://github.com/rwth-i6/returnn/wiki/PyCharm-Configuration

Might help you with this

Reply:
I corrected the emtpy lines.  Thank you Nick, I adjusted my project configuration.